patches/frescobaldi-fix-invalid-metainfo-file.patch

```diff
@@ -0,0 +1,201 @@
+-        </ul>
+-      </description>
+-    </release>
+-    <release version="3.1.0" date="2019-12-27">
+-      <description>
+-        <p>Requirement changes:</p>
+-        <ul>
+-          <li>Frescobaldi now requires Python3.3+</li>
+-        </ul>
+-        <p>New features:</p>
+-        <ul>
+-          <li>New "Document Fonts" dialog supporting text and music fonts and providing a font sample previewer.</li>
+-          <li>Possibility to load external extensions.</li>
+-          <li>New "First System Only" option in Custom Engrave.</li>
+-          <li>New "Goto Line" and "Rename file" commands.</li>
+-          <li>Music View: view settings are remembered per-document.</li>
+-          <li>Music View: copy to Image can copy/export to SVG, PDF and EPS in addition to PNG/JPG.</li>
+-          <li>Music View: new commands to rotate left/right; new toolbar button to show/hide the magnifier.</li>
+-          <li>Music View: new preferences Horizontal/Vertical and Continuous/non-continuous.</li>
+-          <li>Music View: new raster layout mode (displays as many pages as fit in a View).</li>
+-          <li>Manuscript viewer: new toolbar buttons to rotate the pages left/right and to show/hide the magnifier.</li>
+-        </ul>
+-        <p>Bug fixes:</p>
+-        <ul>
+-          <li>Fixed seeking in MIDI player during playing stops sound.</li>
+-          <li>Paper orientation is now properly handled in New Score Wizard.</li>
+-          <li>Fixed discrepancy of LilyPond vs. system version of GhostScript
+-    on Linux.</li>
+-          <li>Fixed includepaths on Windows.</li>
+-          <li>Fixed the error NameError: name 'widgets' is not defined.</li>
+-        </ul>
+-        <p>Improvements:</p>
+-        <ul>
+-          <li>Score Wizard: Titles and Headers are now shown in preview; new checkbox to write/omit pitches after \relative command; allow "none" for instrument names on first system.</li>
+-          <li>Smarter behaviour of the autocompletion popup (#918, #922).</li>
+-          <li>New command File->Rename/Move.</li>
+-          <li>Sessions can be grouped in the Sessions menu.</li>
+-          <li>Show absolute path of include files in tooltip.</li>
+-          <li>Restructure Tools Menu.</li>
+-          <li>File Open toolbar button shows recent files menu on long click.</li>
+-          <li>Added "Blank Sheet Music" template snippet.</li>
+-        </ul>
+-      </description>
+-    </release>
+-    <release version="3.0.0" date="2017-02-17">
+-      <description>
+-        <p>Requirement changes:</p>
+-        <ul>
+-          <li>Frescobaldi now requires Python3.2+, Qt5, PyQt5, python-poppler-qt5.</li>
+-        </ul>
+-        <p>New features:</p>
++        <p>This release contains several bugfixes for Mac OS X.</p>
++        <p>Full history and credits for this release are available at https://github.com/frescobaldi/frescobaldi/compare/v3.1.1...v3.1.2</p>
```

Comment:
This is the URL here that appstream validation do not like.

Reply:
Here is the patch:

```diff
diff --git a/patches/frescobaldi-fix-invalid-metainfo-file.patch b/patches/frescobaldi-fix-invalid-metainfo-file.patch
index 8a8bf2a..ee4860b 100644
--- a/patches/frescobaldi-fix-invalid-metainfo-file.patch
+++ b/patches/frescobaldi-fix-invalid-metainfo-file.patch
@@ -112,6 +112,7 @@ index 069d84c2..26e28d40 100644
 -    </release>
 -    <release version="3.1.1" date="2020-01-04">
 +    <release type="stable" version="3.1.2" date="2020-04-13T00:00:00Z">
++      <url>https://github.com/frescobaldi/frescobaldi/compare/v3.1.1...v3.1.2</url>
        <description>
 -        <p>Bugfixes:</p>
 -        <ul>
@@ -170,7 +171,6 @@ index 069d84c2..26e28d40 100644
 -        </ul>
 -        <p>New features:</p>
 +        <p>This release contains several bugfixes for Mac OS X.</p>
-+        <p>Full history and credits for this release are available at https://github.com/frescobaldi/frescobaldi/compare/v3.1.1...v3.1.2</p>
 +        <p>This release adds the following feature:</p>
          <ul>
 -          <li>Zoom with pinch gesture in Music View.</li>
```

Reply:
Ok, thanks. I've pushed a commit to fix it